db/seeds/playbook.seeds.rb

```diff
@@ -1,136 +1,138 @@
 on when I joined this outfit. It pays well, and the
 benefits are good. Like they say “You don’t have
-to be crazy to work here, but it sure helps!”',
-},
-{
-  name: 'The Spell-Slinger',
-  description: 'Fight fire with fire magic.',
-},
-{
-  name: 'The Spooky',
-  description: 'I can do things, things that normal people can’t.
+to be crazy to work here, but it sure helps!”'
+   },
+   {
+     name: 'The Spell-Slinger',
+     description: 'Fight fire with fire magic.'
+   },
+   {
+     name: 'The Spooky',
+     description: 'I can do things, things that normal people can’t.
 But there’s a price—I haven’t paid it in full, yet,
 but the bill’s gonna come due soon. It’s best I don’t
-tell you any more. You get too close, you’ll get hurt.',
-},
-{
-  name: 'The Wronged',
-  description: 'They took my loved ones. Back then I wasn’t
+tell you any more. You get too close, you’ll get hurt.'
+   },
+   {
+     name: 'The Wronged',
+     description: 'They took my loved ones. Back then I wasn’t
 strong enough to fight, but I studied, trained, and
 now I’m ready to cleanse the world of their taint.
-I’ll kill them all. That’s all I have left.',
-},
-{
-  name: 'The Gumshoe',
-  description: 'You won’t understand this… When I take a case,
+I’ll kill them all. That’s all I have left.'
+   },
+   {
+     name: 'The Gumshoe',
+     description: 'You won’t understand this… When I take a case,
   I’m supposed to do something about it. You’re supposed to do
     something about it whether you like it or not. You’ve got
     to pay for what you’ve done, sweetheart, whatever it is I
     might feel about you. Yes, I’ll have some bad nights, but
-    I’ll still have myself.',
-},
-{
-  name: 'The Hex',
-  description: 'I didn’t have magic fall into my lap. I’m not
+    I’ll still have myself.'
+   },
+   {
+     name: 'The Hex',
+     description: 'I didn’t have magic fall into my lap. I’m not
   blessed, I’m not one of the scary children—I’m just a girl
   who found a way to give herself the strength to fight this
-  war. I don’t have the option of not taking this risk .',
-},
-{
-  name: 'The Pararomantic',
-  description: 'I am never alone so long as we have each other.',
-},
-{
-  name: 'The Searcher',
-  description: 'There’s still so much to be discovered and
+  war. I don’t have the option of not taking this risk .'
+   },
+   {
+     name: 'The Pararomantic',
+     description: 'I am never alone so long as we have each other.'
+   },
+   {
+     name: 'The Searcher',
+     description: 'There’s still so much to be discovered and
   explained, even now. Perhaps only one event in a thousand is
-  true weirdness — but I’ll investigate them all to find it.',
-}].each do |playbook|
-  Playbook.find_or_initialize_by(name: playbook[:name]).update!(playbook)
-end
+  true weirdness — but I’ll investigate them all to find it.'
+   }].each do |playbook|
+    Playbook.find_or_create_by(name: playbook[:name]).update!(playbook)

```

Comment:
Why was this changed to `find_or_create_by`? I don't think the playbook is valid on creation, because the description isn't included in the attributes.

Reply:
Reverting and will revisit once new gear seeding is completed.